scipy/stats/_mstats_basic.py

```diff
@@ -2551,7 +2551,7 @@ def _winsorize1D(a, low_limit, up_limit, low_include, up_include,
                                    upinc, contains_nan, nan_policy)
 
 
-def moment(a, moment=1, axis=0):
+def moment(a, moment=1, axis=0, *, central=None):
```

Comment:
"central" is an adjective, which suggests that the user would pass a boolean (e.g. "yes, I want the central moment"). 

I suggested the noun, "center". The user passes the center about which the moment is taken. (e.g. "take the moment about 0").

In any case, this should be a noun of some sort if it expects the user to pass a real number.